src/core/commands.js

```diff
@@ -586,6 +586,14 @@ export function customTextBetween (slice, from, to, blockSeparator, leafText) {
 		if (node.type === schema.nodes.citation) {
 			text += serializeCitationInnerText(node);
 		}
+		else if (node.type === schema.nodes.math_display) {
+			text += blockSeparator + '$$' + node.textContent + '$$';
+			return false;
+		}
+		else if (node.type === schema.nodes.math_inline) {
+			text += '$' + node.textContent + '$';
+			return false;
+		}
```

Comment:
If I try to copy two paragraphs (one with inline math, and another with just text) and paste into a plain text editor, they appear in a single line.